File: C#JAVA_Client_Server/ClientServcerC#Protocols/LongExample/Domain/src/main/java/entities/domain/ParticipantiCursa.java
package entities.domain;

public class ParticipantiCursa extends Entitate<Integer> {
    Integer idSofer;
    Integer idCursa;

    @Override
    public String toString() {
        return "entities.domain.ParticipantiCursa{" +
                "idSofer=" + idSofer +
                ", idCursa=" + idCursa +
                '}';
    }

    public Integer getIdSofer() {
        return idSofer;
    }

    public void setIdSofer(Integer idSofer) {
        this.idSofer = idSofer;
    }

    public Integer getIdCursa() {
        return idCursa;
    }

    public void setIdCursa(Integer idCursa) {
        this.idCursa = idCursa;
    }

    public ParticipantiCursa(Integer idSofer, Integer idCursa) {
        this.idSofer = idSofer;
        this.idCursa = idCursa;
    }
}

File: C#JAVA_Client_Server/ClientServcerC#Protocols/Solution1/Domain/ParticipantiCursa.cs
using System;

namespace Domain
{
    [Serializable]
    public class ParticipantiCursa : IObject<int>
    {
        private int idSofer;
        private int idCursa;

        public int IdSofer
        {
            get => idSofer;
            set => idSofer = value;
        }

        public int IdCursa
        {
            get => idCursa;
            set => idCursa = value;
        }

        

        public override string ToString()
        {
            return base.ToString()+" "+idSofer.ToString()+" "+idCursa.ToString();
        }

        public ParticipantiCursa(int idSofer, int idCursa)
        {
            this.idSofer = idSofer;
            this.idCursa = idCursa;
        }

        public ParticipantiCursa()
        {
            
        }

        public int Id { get; set; }
    }
}